fifteen_min/src/isochrone.rs

```diff
@@ -101,16 +101,35 @@ impl Isochrone {
             }
         }
 
+        // Generate polygons covering the contour line where the cost in the grid crosses these
+        // threshold values.
+        let thresholds = vec![
+            0.1,
+            Duration::minutes(5).inner_seconds(),
+            Duration::minutes(10).inner_seconds(),
+            Duration::minutes(15).inner_seconds(),
+        ];
+
+        // And color the polygon for each threshold
+        let colors = vec![
+            Color::GREEN.alpha(0.5),
+            Color::ORANGE.alpha(0.5),
+            Color::RED.alpha(0.5),
+        ];
+
         let mut i = Isochrone {
             start,
             options,
             draw: Drawable::empty(ctx),
-            time_to_reach_building,
+            thresholds: thresholds.clone(),
+            colors: colors.clone(),
+            time_to_reach_building: time_to_reach_building.clone(),
             amenities_reachable,
             population,
             onstreet_parking_spots,
         };
-        i.draw = i.draw_isochrone(app).upload(ctx);
+
+        i.draw = draw_isochrone(app, &time_to_reach_building, &thresholds, &colors).upload(ctx);
```

Comment:
Ah, instead of the `clone()`s above and using these variables, we could just do `&i.time_to_reach_building, `&i.thresholds, &i.colors` here. We moved those 3 things into the `Isochrone` struct, but we can still borrow them to call this function

Reply:
That worked! Makes sense now, my next commit will do this for both Isochrone and BorderIsochrone.